ipn/ipnlocal/peerapi.go

```diff
@@ -1184,7 +1184,7 @@ func newFakePeerAPIListener(ip netip.Addr) net.Listener {
 // even if the kernel isn't cooperating (like on Android: Issue 4449, 4293, etc)
 // or we lack permission to listen on a port. It's okay to not actually listen via
 // the kernel because on almost all platforms (except iOS as of 2022-04-20) we
-// also intercept netstack TCP requests in to our peerapi port and hand it over
+// also intercept netstack TCP requests into our peerapi port and hand it over
```

Comment:
I'd revert this one too. Or if it bugs you, rewrite as "also intercept incoming netstack TCP requests to our ..."